app/controllers/home_controller.rb

```diff
@@ -1,5 +0,0 @@
-class HomeController < ApplicationController
-  def index
-
-  end
-end
```

Comment:
Interesting. This should of been removed on a previous merge. Not sure how this snuck back in